Screenbox/Controls/MediaListViewItem.xaml

```diff
@@ -52,63 +52,60 @@
-                <StaticResource x:Key="SubtleButtonBackgroundPressed" ResourceKey="HyperlinkButtonBackgroundPressed" />
-                <StaticResource x:Key="SubtleButtonForeground" ResourceKey="AccentTextFillColorPrimaryBrush" />
-                <StaticResource x:Key="SubtleButtonForegroundPointerOver" ResourceKey="HyperlinkButtonForegroundPointerOver" />
-                <StaticResource x:Key="SubtleButtonForegroundPressed" ResourceKey="HyperlinkButtonForegroundPressed" />
-            </Button.Resources>
-        </Button>
+        <Grid Grid.Column="1">
+            <TextBlock
+                x:Name="TrackNumberText"
+                Padding="0,0,8,0"
+                HorizontalAlignment="Right"
+                VerticalAlignment="Center"
+                Foreground="{ThemeResource TextFillColorTertiaryBrush}"
+                IsHitTestVisible="False"
+                Style="{StaticResource CaptionTextBlockStyle}"
+                Text="{Binding TrackNumberText, FallbackValue=''}"
+                TextWrapping="NoWrap"
+                Visibility="{x:Bind IsTrackNumberVisible}" />
+            <FontIcon
+                x:Name="ItemMediaTypeIcon"
+                HorizontalAlignment="Center"
+                VerticalAlignment="Center"
+                Glyph="{Binding Converter={StaticResource MediaGlyphConverter}}"
+                IsHitTestVisible="False"
+                Visibility="{x:Bind IsIconVisible}" />
+            <Button
+                x:Name="PlayButton"
+                Width="32"
+                Height="32"
+                Padding="0"
+                VerticalAlignment="Center"
+                Command="{x:Bind PlayCommand, Mode=OneWay}"
+                CommandParameter="{Binding}"
+                Style="{StaticResource SubtleButtonStyle}"
+                ToolTipService.ToolTip="{Binding IsPlaying, Converter={StaticResource BoolToPlayPauseTextConverter}}"
+                Visibility="Collapsed">
+                <FontIcon Glyph="{Binding IsPlaying, Converter={StaticResource PlayPauseGlyphConverter}}" />
+                <Button.Resources>
+                    <StaticResource x:Key="SubtleButtonBackground" ResourceKey="HyperlinkButtonBackground" />
+                    <StaticResource x:Key="SubtleButtonBackgroundPointerOver" ResourceKey="HyperlinkButtonBackgroundPointerOver" />
+                    <StaticResource x:Key="SubtleButtonBackgroundPressed" ResourceKey="HyperlinkButtonBackgroundPressed" />
+                    <StaticResource x:Key="SubtleButtonForeground" ResourceKey="AccentTextFillColorPrimaryBrush" />
+                    <StaticResource x:Key="SubtleButtonForegroundPointerOver" ResourceKey="HyperlinkButtonForegroundPointerOver" />
+                    <StaticResource x:Key="SubtleButtonForegroundPressed" ResourceKey="HyperlinkButtonForegroundPressed" />
+                </Button.Resources>
+            </Button>
+        </Grid>
```

Comment:
Adding an extra layer of Grid may negatively affect the performance, especially when this control is intended to be used in a ListView. I'm not sure how much it affects but my rule of thumb has always been to keep the XAML tree as flat as possible.

Reply:
I actually agree with keeping the tree as flat as possible, do I remove it?

Reply:
Yes, please